ansible/roles_ocp_workloads/ocp4_workload_showroom/tasks/report-variables.yaml

```diff
@@ -0,0 +1,31 @@
+- name: Get the Showroom route
+  kubernetes.core.k8s_info:
+    api_version: route.openshift.io/v1
+    kind: Route
+    name: "showroom-proxy"
+    namespace: "{{ _showroom_namespace }}"
+  register: r_showroom_route
+  failed_when: r_showroom_route.resources | default([]) | length != 1
+  until: r_showroom_route is successful
+  retries: 10
+  delay: 5
+
+- name: Output agnosticd_user_info for showroom
+  ansible.builtin.debug:
+    msg:
+      - "Lab instructions: {{ _showroom_url }}"
+      - "showroom_url: {{ _showroom_url }}"
+  vars:
+    _route: "{{ r_showroom_route.resources[0] }}"
+    _showroom_url: >-
+      {{ 'https' if 'tls' in _route.spec else 'http' }}://{{ _route.spec.host }}/
```

Comment:
Showroom should always be https, no?

Reply:
It's code I stole from your bookbag role. :)